packages/cli/package.json

```diff
@@ -29,6 +29,17 @@
     "p-limit": "^4.0.0",
     "strip-indent": "^4.0.0"
   },
+  "exports": {
+    "source": "./src/cli.ts",
+    "link": "./lib/commands/link.js",
+    "build": "./lib/commands/build.js",
+    "sync": "./lib/commands/sync.js"
+  },
```

Comment:
I'm not sure if we need any exports at all

Reply:
We actually don't need them. Add as it is mentioned in discord. Will remove them then